metaflow/datastore/task_datastore.py

```diff
@@ -750,6 +762,33 @@ def save_logs(self, logsource, stream_data):
                 to_store_dict[n] = data
         self._save_file(to_store_dict)
 
+    @require_mode("d")
+    def scrub_logs(self, logsources, stream, attempt_override=None):
+        path_logsources = {
+            self._metadata_name_for_attempt(
+                self._get_log_location(s, stream),
+                attempt_override=attempt_override,
+            ): s
+            for s in logsources
+        }
+
+        # Legacy log paths
+        legacy_log = self._metadata_name_for_attempt(
+            "%s.log" % stream, attempt_override
+        )
+        path_logsources[legacy_log] = stream
+
+        paths = path_logsources.keys()
+
+        deleted_paths = self._delete_file(paths, add_attempt=False)
```

Comment:
do we need to delete? can't we just overwrite?

Reply:
turns out yes, we can simply overwrite. I cleaned up all the delete related additions to the datastore now, instead relying on `save_bytes(..., allow_overwrite=True)` to do the heavy lifting